Gemfile

```diff
@@ -9,7 +9,8 @@ gem 'berkshelf'
 gem 'chefspec'
 gem 'foodcritic'
 gem 'rake'
-gem 'rubocop'
+gem 'rubocop', '~> 1.46'
+gem 'rexml', '~> 3.2', '>= 3.2.5'
```

Comment:
if specifying rubocop version fix the problem then we can omit rexml gem here

Reply:
rubocop dependency has not directly help in rexml version. so mentioned the version in metadata.db only.